vispy/visuals/ellipse.py

```diff
@@ -65,14 +68,29 @@ def _generate_vertices(self, pos, radius, start_angle, span_angle,
                                                              len(radius)))
         else:
             xr = yr = radius
-        curve_segments = int(num_segments * span_angle / 360.)
-        start_angle *= (np.pi/180.)
-        self._vertices = np.empty([curve_segments+2, 2], dtype=np.float32)
-        self._vertices[0] = np.float32([pos[0], pos[1]])
-        theta = np.linspace(start_angle, start_angle + (span_angle/180.)*np.pi,
-                            curve_segments+1)
-        self._vertices[1:, 0] = pos[0] + xr * np.cos(theta)
-        self._vertices[1:, 1] = pos[1] + yr * np.sin(theta)
+
+        # divide the total sweeped angle into curve_segments
+        # so that the _total_ curve has num_segments, and a
+        # curve of span_ange < 360 will have that fraction
+        # of num_segments
+        # curve_segments = int(num_segments * span_angle / 360.)
```

Comment:
I don't know if this is just me, but I found the fact that ```num_segments``` maps to 360 degrees, and that the ```EllipseVisual``` internally rescales the ```num_segments``` to scale down to whatever angle you pass to be very unintuitive.


eg:

If I pass ```num_segments=10, span_angle=180```, In the old implementation, the ellipse would have ```5``` segments.

I changed so that if ```num_segments=10, span_angle=180```, then the 180 degree arc is drawn with ```10``` segments, which I found to be more intuitive.

Do give me feedback on this. I'll revert this if needed